src/include/duckdb/common/bitpacking.hpp

```diff
@@ -178,7 +178,31 @@ class BitpackingPrimitives {
 	}
 
 	template <class T>
-	static void UnPackGroup(data_ptr_t dst, data_ptr_t src, bitpacking_width_t width,
+	static void ApplyFrameOfReference(data_ptr_t dst, T frame_of_reference) {
+		if (!frame_of_reference) {
+			return;
+		}
+		for (idx_t i = 0; i < BITPACKING_ALGORITHM_GROUP_SIZE; i++) {
+			T value = Load<T>(dst + (i * sizeof(T)));
```

Comment:
Do we need Load and Store here? I think the dst here should always be aligned, no?

Reply:
Yep you're right, changed it